stable/rabbitmq/values-production.yaml

```diff
@@ -125,6 +125,8 @@ service:
   ## ref: https://github.com/bitnami/bitnami-docker-rabbitmq#environment-variables
   ##
   managerPort: 15672
+  annotations:
```

Comment:
Add a comment to the annotation property.
Also, leave a blank line between managerPort and the annotation comment. 

Please do the same changes in the values.yaml file too